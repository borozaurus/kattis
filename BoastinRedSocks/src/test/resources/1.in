1 2
6 8
12 2499550020
56 789
0 06