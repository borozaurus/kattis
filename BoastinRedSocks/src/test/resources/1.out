3 1
7 1
4 49992
impossible